code/modules/mob/living/carbon/human/human.dm

```diff
@@ -455,30 +455,34 @@
 			return FALSE
 
 		if (target.stat == DEAD || HAS_TRAIT(target, TRAIT_FAKEDEATH))
-			to_chat(src, span_warning("[target.name] is dead!"))
+			balloon_alert(src, "[target.name] is dead!")
 			return FALSE
 
 		if (is_mouth_covered())
-			to_chat(src, span_warning("Remove your mask first!"))
+			balloon_alert(src, "remove your mask first!")
 			return FALSE
 
 		if (target.is_mouth_covered())
-			to_chat(src, span_warning("Remove [p_their()] mask first!"))
+			balloon_alert(src, "remove [p_their()] mask first!")
 			return FALSE
 
-		if (!get_organ_slot(ORGAN_SLOT_LUNGS))
-			to_chat(src, span_warning("You have no lungs to breathe with, so you cannot perform CPR!"))
+		if(HAS_TRAIT_FROM(src, TRAIT_NOBREATH, DISEASE_TRAIT))
+			to_chat(src, span_warning("you can't breathe!"))
```

Comment:
```suggestion
			to_chat(src, span_warning("you don't breathe!"))
```

`can't` implies that something is _wrong_, when self-resp is just an alternative to what is correct.

Reply:
The original premise of the PR was that the virus allowed you to breathe if you wanted to, thus enabling CPR for carriers of it; however, other maintainers have stated that the virus should cause a *complete inability* to breathe. The current wording of "can't" is logically correct if the other maintainer instructions are to be followed.

That said, I can still see the sentiment of using less harsh/alarming wording. Please let me know if you would still like it to be changed.

Reply:
Alright, just wanted to make sure the User Feedback was correct